deploy/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM nypl/circ-base
+
+MAINTAINER Library Simplified <info@librarysimplified.org>
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        nginx \
+        supervisor
+
+# Set up Nginx & UWSGI
+RUN rm /etc/nginx/sites-enabled/default
+COPY nginx.conf /etc/nginx/conf.d/circulation.conf
+RUN echo "daemon off;" >> /etc/nginx/nginx.conf
```

Comment:
Why modify the core `nginx.conf` file instead of putting `daemon off` into the custom `nginx.conf` you copy to `/etc/nginx/conf.d/`?